linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/rule.yml

```diff
@@ -26,6 +27,10 @@ ocil: |-
     following command:
     <pre>$ update-crypto-policies --show</pre>
     The output should return <pre><sub idref="var_system_crypto_policy" /></pre>.
+    Moreover, check if crypto policy customizations are applied correctly.
+    <pre>$ ls -l /etc/crypto-policies/back-ends/</pre>
+    For every file, check if it is a symbolic link or a regular file. If it is a symbolic link, verify that the target resides in the <pre>/usr/share/crypto-policies/<sub idref="var_system_crypto_policy" />/</pre> directory. If it is a regular file, check that there exists a drop in file in the <pre>/etc/crypto-policies/local.d</pre>. The file name of a drop in file must be formed as <pre>LIBRARY_NAME-ARBITRARY_ID.config</pre>.
+    For example, the drop in file for Openssh server can be named <pre>opensshserver-ospp.config</pre>.
```

Comment:
Make this more direct, with commands and expected output.
For example, command below will give any file which is not a symlink:
```
ls -l /etc/crypto-policies/back-ends | grep "^[^lt]"
-rw-r--r--. 1 root root 429 Jul  9 14:16 nss.config
```
And if any result is returned, check that there are files in `/etc/crypto-policies/local.d` whose name match the libraries which is not symlink'ed.
`ls /etc/crypto-policies/local.d/ | awk -F'-' '{print $1}'` will print only the library names in `local.d` directory.